src/main/scss/components/_buttons-deprecated.scss

```diff
@@ -109,209 +109,3 @@
 .button-disabled {
   @include button-disabled;
 }
-
-// Button styles
-
-.yui-button {
```

Comment:
we might have the one or other plugin that use the construct
```
<span class="yui-button">
  <button ...>
</span>
```
Those might look broken when we remove all the yui styling here.